src-java/kilda-model/src/main/java/org/openkilda/model/Switch.java

```diff
@@ -123,6 +130,11 @@ public Switch(@NonNull SwitchId switchId, SwitchStatus status, InetSocketAddress
         this.timeModify = timeModify;
         setFeatures(features);
         this.pop = pop;
+        this.latitude = latitude;
+        this.longitude = longitude;
+        this.street = street;
+        this.city = city;
+        this.country = country;
```

Comment:
Should we put these 5 fields into `SwitchLocation` class too?

Reply:
No, we shouldn't, from db perspective it's a straightfowrward node attributes without grouping

Reply:
We discussed that we should store these fields directly on the switch in the database.